contributed_definitions/NXxpcs.nxdl.xml

```diff
@@ -57,15 +57,31 @@
 
     <field name="entry_identifier">
       <doc>
-        Unique identifier for the experiment.
+        **Locally** unique identifier for the experiment (a.k.a. run or scan).
+
+        * For bluesky users, this is the run's `"scan_id"`.
+        * For SPEC users, this is the scan number (``SCAN_N``).
       </doc>
     </field>
     <field name="entry_identifier_uuid" minOccurs="0" maxOccurs="1">
       <doc>
         (optional) UUID identifier for this entry.
+
+        See (standard)[https://www.rfc-editor.org/rfc/rfc4122.html] or
+        (wikipedia)[https://en.wikipedia.org/wiki/Universally_unique_identifier]
+
+        * For bluesky users, this is the run's `"uid"` and is expected
+          for that application.
```

Comment:
Reading 73 and 74 now I was confused (and I was the one that typed this).

The point is we expect people writing NXxpcs to include "entry_identifier_uuid" so maybe this is better.  @prjemian I am happy for you to make the final call.
```
* For bluesky users, this is the run's `"uid"` and this field is expected
           for bluesky users/developers.
```

Reply:
And do the links compile correctly.  I stupidly typed what I thought was correct and didn't see the  link generate in GitHub so I altered it.  

Later, I realized it was in markdown so it would never show the link.  Is the correct for mat `[](url)`?

Reply:
Good.  ESRF was also interested in using a UUID like this. They don't use Bluesky.

Reply:
I guess you refer to https://github.com/nexusformat/definitions/issues/1043

We have a UUID for the sample and a UUID for the "dataset" (refers to on ICAT dataset, not to be confused with an HDF5 dataset of nexus field) to which the NXentry belongs (1 scan - 1 NXentry).

The NXentry itself has a scan number. So at the ESRF an NXentry in our raw data files is uniquely defined by the combination of a "dataset" UUID and a scan number.